base/server/etc/default.cfg

```diff
@@ -150,19 +150,23 @@ pki_ssl_server_subject_dn=cn=%(pki_hostname)s,ou=%(pki_instance_name)s,o=%(pki_s
 pki_ssl_server_token=
 
 pki_sslserver_key_algorithm=%(pki_ssl_server_key_algorithm)s
+pki_sslserver_signing_algorithm=SHA256withRSA
```

Comment:
why is this one fixed to SHA256withRSA?

Reply:
That's just the default value, which all the other similar signing alg variants have been set to.

Reply:
Also, cfu, the ones you see above are defaulting to deprecated versions of the alg. Since this one is added there is nothing to deprecate.